ios_app_clip/lib/main.dart

```diff
@@ -0,0 +1,132 @@
+import 'package:device_info/device_info.dart';
+import 'package:flutter/material.dart';
+
+void main() {
+  runApp(MyApp());
+}
+
+class MyApp extends StatelessWidget {
+  // This widget is the root of your application.
```

Comment:
I would recommend cutting all of these comments rom the template. Also, consider stripping this down to just the plugin output and some kind of graphic (like a spinning Flutter logo or something).

Reply:
Ah ok, SG. Done